CHANGELOG.md

```diff
@@ -1,5 +1,28 @@
 # Change Log
 
+## [0.13.0](https://github.com/eclipse/lemminx/milestone/16?closed=1) (July 1, 2020)
+
+### Enhancements
+
+ * Grammar generator: generate a grammar from an XML document. See [#778](https://github.com/eclipse/lemminx/issues/778).
+ * Improved schema generation from an XML document whose name is also an extension. See [#805](https://github.com/eclipse/lemminx/issues/805).
+ * DTD generation: support for "optional" elements. See [#789](https://github.com/eclipse/lemminx/issues/789).
+ * Multiplicity for repeated elements in generated .XSD. See [#788](https://github.com/eclipse/lemminx/issues/788).
+ * Improved attribute generation for the grammar generator. See [#791](https://github.com/eclipse/lemminx/issues/791).
+ * Bind XML document with no grammar constraints to generated XSD / DTD. See [#151](https://github.com/eclipse/lemminx/issues/151).
+ * Quick fix to create missing `xsi:noNamespaceSchemaLocation` and generate XSD that adheres to current XML document. See [#702](https://github.com/eclipse/lemminx/issues/702).
+ * Highlight the XSD file name in `xsi:schemaLocation` when reporting an invalid or missing XSD file. See [#782](https://github.com/eclipse/lemminx/issues/782).
+ * Add support for `textDocument/documentLink` for `xsi:schemaLocation`. See [#666](https://github.com/eclipse/lemminx/issues/666).
+ * Sort snippets. See [#692](https://github.com/eclipse/lemminx/issues/692).
+ * Formatting support for trim trailing whitespace. See [vscode-xml#250](https://github.com/redhat-developer/vscode-xml/issues/250).
+ * Warning message when one of the `xml.catalogs` paths cannot be found. See [vscode-xml#217](https://github.com/redhat-developer/vscode-xml/issues/217).
```

Comment:
Please use the lemminx link: https://github.com/eclipse/lemminx/pull/757. Sorry, I didn't add the lemminx PR to the milestone. It's added now.